hash/thinkpad8xx.xml

```diff
@@ -14,33 +14,84 @@ license:CC0-1.0
 		<sharedfeat name="compatibility" value="800,820,850" />
 		<part name="cdrom1" interface="cdrom">
 			<feature name="part_id" value="AIX 4 Volume 1 of 2" />
+			<feature name="form" value="LK2T0241" /> <!-- FORM on internal .Version -->
+			<feature name="date" value="970227070036" /> <!-- September 29, 2000, on internal .Version -->
 			<diskarea name="cdrom">
 				<disk name="aix_4.1.5_disc_01" sha1="4c49c8190e33c2c7f13a74d7332d384edd6b3042" />
 			</diskarea>
 		</part>
 		<part name="cdrom2" interface="cdrom">
 			<feature name="part_id" value="AIX 4 Volume 2 of 2" />
+			<feature name="date" value="970331130110" /> <!-- September 30, 2000, on internal .Version  -->
```

Comment:
Are you sure these are supposed to be UNIX timestamps?  They don’t make sense interpreted that way.  Wouldn’t it be more likely to be 1997-03-31T13:01:10?

Reply:
Yes, absolutely. I've fixed it.